cam_rad_fusion:
  # Topics
  imageTopic: "/camera_array/cam0/image_raw"
  lidarTopic: "/ouster/points"
  radarTopic: "/point_cloud"

  # Save
  saveDirectory: "/home/ysh/catkin_ws/src/cam_rad_fusion/DATA/"
  save_count_duty: 5